### ISSUE TITLE
chmod has wrong mode

### ISSUE THREAD

**Issue by aporzio1** [mudmin/AnotherMaker#2]
In the guide you have "Make script executable (chmod **x+** /usr/local/bin/opencanary.sh)" but it should be "(chmod **+x** /usr/local/bin/opencanary.sh)"

**Comment by Nyrup** [comment]
> In the guide you have "Make script executable (chmod **x+** /usr/local/bin/opencanary.sh)" but it should be "(chmod **+x** /usr/local/bin/opencanary.sh)"

I noticed that as well. Did you manage to get it auto-started at boot? I fail this last part but everything seems to work fine if I start the canary manually....

**Comment by mbuhrley** [comment]
Yes, simple fix on the chmod issue +x but I can't get auto-start working according to those instructions either. Works fine with manual start.

**Comment by Nyrup** [comment]
> Yes, simple fix on the chmod issue +x but I can't get auto-start working according to those instructions either. Works fine with manual start.

[This guide](https://simpaul.com/open-canary-on-a-pi/) worked for me using Raspberry PI OS Desktop version (including the auto start service)
